scripts/setup_dev_data.sh

```diff
@@ -154,19 +154,21 @@ ${local_aws} ssm put-parameter --name /genepi/local/localstack/nextstrain-sfn --
   "RunSPOTVcpu":10,
   "StateMachineArn":'${LOCAL_SFN_ARN}'
 }'
-${local_aws} ssm delete-parameter --name /genepi/local/localstack/pangolin-ondemand-sfn
```

Comment:
This commit chunk looks weird and wrong -- why is pangolin changing into lineage-qc, we need both! -- but it's really just two different pieces of work smushed together.

1. Added in a new local SSM param for doing lineage-qc
2. I noticed that we had duplicated `pangolin-ondemand-sfn` local SSM params. There were two of them, with the second simply clobbering the first, meaning the first was doing nothing in the loading script. I chatted a bit with Jess about it, it seems like the first is probably a better representation of real SSM param at this point, so I deleted this one, the second one. The second actually was what we were deving against since it was doing the clobbering, but I tested it locally and there's no issues.